src/NServiceBus.Core/Outbox/OutboxDeduplicationBehavior.cs

```diff
@@ -63,23 +58,35 @@ void DispatchOperationToTransport(IEnumerable<TransportOperation> operations)
 
                 deliveryOptions.EnlistInReceiveTransaction = false;
 
-                var message = new OutgoingMessage(transportOperation.MessageId,transportOperation.Headers, transportOperation.Body);
+                var message = new OutgoingMessage(transportOperation.MessageId, transportOperation.Headers, transportOperation.Body);
 
-                //dispatch to transport
+                var operationType = transportOperation.Options["Operation"];
```

Comment:
So instead of polymorphic dispatch we have now a stringly type way to dispatch. Not sure I like it.